docs/environment-variables.md

```diff
@@ -13,6 +13,7 @@ Note that these environment variables may be removed at any time.
 | `ARCHIVED_WORKFLOW_GC_PERIOD` | `time.Duration` | The periodicity for GC of archived workflows. |
 | `ARGO_TRACE` | `bool` | Whether to enable tracing statements in Argo components. |
 | `DEFAULT_REQUEUE_TIME` | `time.Duration` | The requeue time for the rate limiter of the workflow queue. |
+| `EXPRESSION_TEMPLATES` | `bool` | Escape hatch to disable expression templates. Default `true`. |
```

Comment:
Hopefully my doc checker worked in this case :)

Reply:
This leaves me a _little_ confused.  The var name seems like a positive, but the description a negative, so i am not sure my assumed used of this would be correct.  Perhaps either: 

```
| `DISABLE_EXPRESSION_TEMPLATES` | `bool` | Escape hatch to disable expression templates. Default `false`. |
```

Or more clarification:

```
| `EXPRESSION_TEMPLATES` | `bool` | If set to `false` (or anything other than `true`) expression templates will be disabled. Default `true`. |
```